src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -127,27 +129,31 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
   }
 
   protected async firstUpdated(): Promise<void> {
-    const loaded = await loadRoundslider();
-
-    this._roundSliderStyle = loaded.roundSliderStyle;
-    this._jQuery = loaded.jQuery;
-
     const stateObj = this.hass!.states[this._config!.entity] as LightEntity;
 
+    if (supportsFeature(stateObj, SUPPORT_BRIGHTNESS)) {
```

Comment:
This is not good, as this means that it's not loaded if the user would change the entity ID (for example in the live editor)

Reply:
Yeah that makes sense